src/utils/refineSearchUtils.ts

```diff
@@ -45,13 +45,24 @@ export const addLabelPropAndParseFilters = (
 ): Record<string, Option[]> => {
   const appliedFilterValuesWithLabels = {}
   for (const appliedFilterField in appliedFilterValues) {
-    // Find the aggregation that corresponds to the filter field we are working on
-    const matchingFieldAggregation = aggregations.find(
-      ({ field: aggregationField }) => aggregationField === appliedFilterField
-    )
     appliedFilterValuesWithLabels[appliedFilterField] = appliedFilterValues[
       appliedFilterField
     ].map((filterValue: string): Option => {
+      // dateBefore and dateAfter fields are not based on
+      // aggregations results. Pass the year along with out
+      // transforming fieldname or finding the label
+      if (appliedFilterField.includes("date")) {
+        const labelPrefix = appliedFilterField.split("date")[1]
+        return {
+          count: null,
+          value: filterValue,
+          label: labelPrefix + " " + filterValue,
```

Comment:
Small nit/preference: I prefer to use string interpolation for this kind of thing.